addons/auth_signup/data/mail_template_data.xml

```diff
@@ -40,11 +40,12 @@
                         Dear <t t-out="object.name or ''">Marc Demo</t>,<br /><br />
                         You have been invited by <t t-out="object.create_uid.name or ''">OdooBot</t> of <t t-out="object.company_id.name or ''">YourCompany</t> to connect on Odoo.
                         <div style="margin: 16px 0px 16px 0px;">
-                            <a t-att-href="object.signup_url"
+                            <a t-att-href="object.partner_id._get_signup_url()"
                                 t-attf-style="background-color: {{object.company_id.email_secondary_color or '#875A7B'}}; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;">
                                 Accept invitation
                             </a>
                         </div>
+                        <b>  This link will remain valid during <t t-out="int(int(object.env['ir.config_parameter'].sudo().get_param('auth_signup.signup.validity.hours',144))/24)"></t> days </b> <br/>
```

Comment:
I think we have tools/libraries to express an interval in human readable form, though this is nitpicking it can be done later on / it should delay the merge of this task.

Reply:
```python
In [13]: self.env['ir.qweb.field.duration'].value_to_html(self.env['ir.config_parameter'].sudo().get_param('auth_signup.signup.validity.hours',144), {'unit': 'hour'})
Out[13]: '6 days'
In [14]: self.env['ir.qweb.field.duration'].value_to_html(self.env['ir.config_parameter'].sudo().get_param('auth_signup.signup.validity.hours',12), {'unit': 'hour'})
Out[14]: '12 hours'
```
which is using `babel.dates.format_timedelta` under the hood. You could also choose to directly use `babel.dates.format_timedelta` but it looks more complicated.